docs/_layouts/default.html

```diff
@@ -12,9 +12,9 @@
 
     {% include nav-home.html %}
 
-    <div id="content">
+    <main id="content" role="main">
```

Comment:
agree, it's unnecessary normally, but notoriously IE11 needs this extra `role` to make it work (as in, expose it correctly to AT). as it doesn't do any harm, this PR looks fine to me as is